source/quickstart.txt

```diff
@@ -295,7 +295,7 @@ Synchronization Notes
 
 - The default port for the HTTP API is ``27182``.  Use the ``--port``
   option with ``mongosync`` to :ref:`configure another port
-  <c2c-mongosync>`
+  <c2c-mongosync-behavior>`
```

Comment:
[question] should this link to the configuration page as opposed to the behavior page?

Reply:
Yes, good catch.